internal/controller/operatorconfigmap_controller.go

```diff
@@ -85,6 +85,18 @@ const (
 	noobaaCrName               = "noobaa-remote"
 )
 
+var (
+	managedPackageNames = []string{
+		"ocs-client-operator",
+		"odf-csi-addons-operator",
+		"csi-addons",
+		"cephcsi-operator",
+		"ceph-csi-operator",
+		"recipe",
+		"odf-external-snapshotter-operator",
+	}
```

Comment:
Why do we need to manually manage package names? any operator within the namespace can be the reason that upgrades are not happening, as such if we are to approve we need to approve all install plans in the namespace regardless of the content of the install plans